test/passwordresettests/passwordResetV1.test.js

```diff
@@ -16,14 +16,14 @@ describe('when handling a password reset (v1)', () => {
   beforeEach(() => {
     send = jest.fn();
 
-    const email = require('./../../src/email');
+    const email = require('../../src/infrastructure/email');
```

Comment:
not sure we want to continue doing the ./ before require here as well?

Reply:
We do, missed them after the auto-refactor. Changed now